src/styles/main.scss

```
@@ -1,3 +1,187 @@
-body {
-  margin: 0;
+@import "all_reset";
+@import "fonts";
+
+.page {
+  &__header {
+    margin-bottom: 50px;
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  height: 60px;
+  padding: 0 50px;
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__item {
+    margin-left: 20px;
+  }
+  
+  &__item:first-child {
+    margin-left: 0;
+  }
```

```suggestion
  &__item {
    margin-left: 20px;
    
    &:first-child {
      margin-left: 0;
    }
  }
```
Це не дуже суттєво, але так код виглядає більш зрозумілим